mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.kt

```diff
@@ -1,114 +1,40 @@
 package org.mifos.mobilewallet.mifospay.faq.ui
 
 import android.os.Bundle
-import android.widget.ExpandableListView
-import org.mifos.mobilewallet.mifospay.R
+import androidx.activity.compose.setContent
+import androidx.activity.viewModels
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableIntStateOf
 import org.mifos.mobilewallet.mifospay.base.BaseActivity
-import org.mifos.mobilewallet.mifospay.faq.FAQContract
-import org.mifos.mobilewallet.mifospay.faq.FAQContract.FAQView
-import org.mifos.mobilewallet.mifospay.faq.FAQListAdapter
-import org.mifos.mobilewallet.mifospay.faq.presenter.FAQPresenter
-import org.mifos.mobilewallet.mifospay.utils.Constants
-import javax.inject.Inject
+import org.mifos.mobilewallet.mifospay.faq.presenter.FAQViewModel
+import org.mifos.mobilewallet.mifospay.theme.MifosTheme
 
 /**
  * This class is the UI component of the Architecture.
  *
  * @author ankur
  * @since 11/July/2018
  */
-class FAQActivity : BaseActivity(), FAQView {
-    @JvmField
-    @Inject
-    var mPresenter: FAQPresenter? = null
-    private var mFAQPresenter: FAQContract.FAQPresenter? = null
-    private var expandableListView: ExpandableListView? = null
-    private var faqListAdapter: FAQListAdapter? = null
-    private var listDataGroup: MutableList<String>? = null
-    private var listDataChild: HashMap<String, List<String>>? = null
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_faq)
-        showColoredBackButton(Constants.BLACK_BACK_BUTTON)
-        setToolbarTitle(Constants.FAQ)
-
-        // initializing the views
-        initViews()
-
-        // initializing the objects
-        initObjects()
-
-        // preparing list data
-        initListData()
-    }
-
-    override fun setPresenter(presenter: FAQContract.FAQPresenter?) {
-        mFAQPresenter = presenter
-    }
-
-    /**
-     * Method to initialize the views
-     */
-    override fun initViews() {
-        expandableListView = findViewById(R.id.faq_list)
-    }
-
-    /**
-     * Method to initialize the objects
-     */
-    override fun initObjects() {
-
-        // initializing the list of groups
-        listDataGroup = ArrayList()
+class FAQActivity : BaseActivity() {
 
-        // initializing the list of child
-        listDataChild = HashMap()
+    private var selectedFaqPosition: MutableState<Int> = mutableIntStateOf(-1)
 
-        // initializing the adapter object
-        faqListAdapter = FAQListAdapter(this, listDataGroup as ArrayList<String>, listDataChild!!)
+    private val faqViewModel: FAQViewModel by viewModels()
```

Comment:
You don't need to initialize the viewmodel here anymore, `hiltViewModel` in compose screen gonna take care.